docs/starting-operator-with-webhooks.md

```diff
@@ -0,0 +1,44 @@
+# Enabling Webhooks in Kyma Operator
+
+To make local testing easier, webhooks are disabled by default. To enable webhooks running with the operator,
+you will have to change some kustomization.yaml files as well as introduce a flag that will enable the webhook server:
+
+
+In `config/crd/kustomization.yaml`:
+
```

Comment:
the related config under `operator/config/default/kustomization.yaml` also need to be uncommented, and cert manager needs to deployed. Maybe mention this tutorial is also an option: https://book.kubebuilder.io/cronjob-tutorial/running-webhook.html